test/extensions/filters/http/rbac/rbac_filter_test.cc

```diff
@@ -238,7 +240,8 @@ class RoleBasedAccessControlFilterTest : public testing::Test {
     *config.mutable_matcher() = matcher;
 
     *config.mutable_shadow_matcher() = matcher;
-    config.set_shadow_rules_stat_prefix("prefix_");
+    config.set_rules_stat_prefix("prefix_");
```

Comment:
Instead of changing the stat_prefix for all tests, can there be a dedicated test where this behavior is validated (the motivation is that the rest of the tests are only targeting other validations, and that the case where "stat_prefix" isn't set is also exercised).

Also please change the value from "prefix_" to "rule_prefix_".

Reply:
I added a separate test purely for `rules_stat_prefix` and left the existing `shadow_rules_stat_prefix` in tests alone (other than renaming the prefix from `prefix_` to `shadow_rules_prefix_`.